### ISSUE TITLE
Pure ground-glass pulmonary nodule

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6637]
**Preferred term label:**Pure ground-glass pulmonary nodule

**Synonyms**
PGGN

**Definition (free text, please give PubMed ID)**
Parenchymal opacification with a higher density than the surrounding tissue, not obscuring underlying bronchovascular structures.

**Parent term (use hpo.jax.org/app)**
Subsolid pulmonary nodule (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
